modules/nf-core/muse/call/meta.yml

```diff
@@ -0,0 +1,63 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "muse_call"
+description: pre-filtering and calculating position-specific summary statistics using
+  the Markov substitution model
+keywords:
+  - genomics
+  - wgs
+  - wxs
+  - vcf
+tools:
+  - "MuSE":
+      description: "Somatic point mutation caller based on Markov substitution model
+        for molecular evolution"
+      homepage: "https://bioinformatics.mdanderson.org/public-software/muse/"
+      documentation: "https://github.com/wwylab/MuSE"
+      tool_dev_url: "https://github.com/wwylab/MuSE"
+      doi: "10.1101/gr.278456.123"
+      licence: ["https://github.com/danielfan/MuSE/blob/master/LICENSE"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1' ]`
+    - tumor_bam:
+        type: file
+        description: Sorted tumor BAM file
+        pattern: "*.bam"
+    - normal_bam:
+        type: file
+        description: Sorted matched normal BAM file
+        pattern: "*.bam"
+  - - meta2:
+        type: map
+        description: |
+          Groovy Map containing reference information.
+          e.g. `[ id:'test' ]`
+    - reference:
+        type: file
+        description: faidx indexed reference genome file
```

Comment:
An indexed file? Does that mean it wants the index to be present alongside?

Reply:
I just copied that from the module description but to me it seems that is also happy with just the fasta file